Dockerfile.gcc

```diff
@@ -93,6 +93,9 @@ ENV KCONFIG_NOTIMESTAMP="true"
 ENV KBUILD_BUILD_TIMESTAMP="${KBUILD_BUILD_TIMESTAMP}"
 ENV SOURCE_DATE_EPOCH="${SOURCE_DATE_EPOCH}"
 
+# ensure CONFIG_BPF_KPROBE_OVERRIDE is not set
+RUN ! grep -q -E 'CONFIG_BPF_KPROBE_OVERRIDE=[y|m]' .config
```

Comment:
It'll be nice to spill a error like "Can't set this option for security reasons".